.github/ISSUE_TEMPLATE/offboarding.md

```diff
@@ -35,26 +35,42 @@ ticket has been completed. At that time, put the ticket back to triage.
 
 ## Project manager actions
 
-- [ ] Nominated [INSERT-NAME-HERE] as a replacement system administrator <sub>or the offboarded employee/contractor was not the sole system administrator</sub>
+- [ ] Nominated [INSERT-NAME-HERE] as a replacement system administrator <sub>or the system administrator is not being replaced</sub>
 - [ ] Assigned ticket to the offboarded employee/contractor
 
 
 ## Offboarded employee/contractor actions
 
-- [ ] Handed over credentials of all GitLab root accounts to the replacement system administrator <sub>or the offboarded employee/contractor was not the sole system administrator</sub>
+- [ ] Assigned the replacement system administrator's account as a maintainer of the `Azul Triagers` team on GitHub <sub>or the system administrator is not being replaced</sub>
+- [ ] Assigned the replacement system administrator's account as a maintainer of the `Azul Developers` team on GitHub <sub>or the system administrator is not being replaced</sub>
+- [ ] Assigned the replacement system administrator's account as a maintainer of the `Azul Operators` team on GitHub <sub>or the system administrator is not being replaced</sub>
+- [ ] Handed over credentials of all GitLab root accounts to the replacement system administrator <sub>or the system administrator is not being replaced</sub>
+- [ ] Created a new `dev` GitLab EC2 instance using a private key from the replacement system administrator <sub>or the system administrator is not being replaced</sub>
+- [ ] Created a new `prod` GitLab EC2 instance using a private key from the replacement system administrator <sub>or the system administrator is not being replaced</sub>
+- [ ] Created a new `anvildev` GitLab EC2 instance using a private key from the replacement system administrator <sub>or the system administrator is not being replaced</sub>
+- [ ] Created a new `anvilprod` GitLab EC2 instance using a private key from the replacement system administrator <sub>or the system administrator is not being replaced</sub>
```

Comment:
I don't think this is accurate. Aren't private keys supposed to be private?
The document reads "A _system administrator_ changes the credentials of all GitLab root accounts.", assuming this is the step this checklist items are covering. But I might be reading this wrong, I differ to your expertise.

Reply:
The [GitLab Own](https://docs.google.com/document/d/1Kg0dMZmCw6gtkvabD2jYWPZO2Mx_wsC8BJPKdeKTfg0/edit#heading=h.ohg7b2fljfxq) role section of the [UCSC Data Browser Security Architecture](https://docs.google.com/document/d/1Kg0dMZmCw6gtkvabD2jYWPZO2Mx_wsC8BJPKdeKTfg0/edit) document has further details about removal of this role.

> Removal of this role from an account requires the creation of a new GitLab EC2 instance, using the private key of another system administrator who will then assign this role to themselves as described in the previous section.

You are correct however that this should be an action taken by the replacement system administrator and not the offboarded employee.